{
  "try": {
    "success": {
      "message": "Запрос к ${url} завершён успешно.",
      "data": "Полученные данные: ${JSON.stringify(data, null, 2)}"
    },
    "error": {
      "message": "Запрос к ${url} завершился с ошибкой.",
      "data": "Содержание ошибки: ${data}"
    }
  },
  "catch": {
    "error": "Сообщение об ошибке:\n${err}"
  }
}
